petergreve/application/test/stores/message-store.js

```diff
@@ -0,0 +1,35 @@
+var rewire = require("rewire");
+var Constants = require('../../app/constants/constants')
+var MessageStore = rewire('../../app/stores/message-store')
+import Dispatcher from '../../app/dispatcher/dispatcher'
+
+describe('Message Store methods', () => {
+	
+	before(() => {
+        MessageStore.__set__('_messages',[])

```

Comment:
I must admit that I adopted this approach too in my store tests :-)

Makes the test setup much cleaner.